cypress-tests/cypress/e2e/spec/Payment/00031-UCSFlowTest.cy.js

```diff
@@ -0,0 +1,184 @@
+import * as fixtures from "../../../fixtures/imports";
+import State from "../../../utils/State";
+import { payment_methods_enabled } from "../../configs/Payment/Commons";
+import getConnectorDetails, * as utils from "../../configs/Payment/Utils";
+
+let globalState;
+
+describe("UCS Flow Testing", () => {
+  const UCS_SUPPORTED_CONNECTORS = ["authorizedotnet"];
```

Comment:
put this in the inclusion list instead: `cypress-tests/cypress/e2e/configs/Payment/Utils.js`